### PR TITLE
Update login_component.html

### PR DESCRIPTION
**Pull request by MeissnerJan**

### CODE REVIEW

File: app_angular/lib/src/auth/login_component.html [hunk 1]
```diff
@@ -2,8 +2,7 @@
     <h2>Please login</h2>
 
     <material-checkbox [disabled]="isAuthenticating" themeColor="#0091fb" [(checked)]="consentChecked"></material-checkbox>
-    <p>I consent to receiving follow-up communication from the Mews recruiting team concerning participation results and
-        potential employment opportunities.<br>Read more <a href="https://www.mewssystems.com/privacy" target="_blank">here</a>.
+    <p>I consent to receiving ad-hoc e-mail communication from the Mews team. You can always request removal of your e-mail at tech.dr@mews.com. <br>Read more <a href="https://www.mews.com/privacy" target="_blank">here</a>.
```
Comment (ookami-kb) [inline app_angular/lib/src/auth/login_component.html]: This sounds a little bit weird for me: "**I** consent... **You** can always request..."

Comment (MeissnerJan) [reply app_angular/lib/src/auth/login_component.html]: good point. Let me re-do it.

**Comment by ookami-kb** [comment]
@MeissnerJan I would maybe also make it smaller and italic, like this:


What do you think?

**Comment by MeissnerJan** [comment]
NICE

**Pull request merged**